WordPress/Classes/ViewRelated/Reader/WPImageViewController.m

```diff
@@ -314,7 +314,7 @@ + (BOOL)isUrlSupported:(NSURL *)url
 
     NSArray *types = @[@".png", @".jpg", @".gif", @".jpeg"];
     for (NSString *type in types) {
-        if (NSNotFound != [absoluteURL rangeOfString:type].location) {
+        if (NSNotFound != [[absoluteURL lowercaseString] rangeOfString:type].location) {
```

Comment:
Is this change related to this PR?

Reply:
Yes, it is. An image I tried upload had a capitalized extension so it failed this test.

Reply:
Good catch!